src/wagmi.ts

```diff
@@ -4,7 +4,10 @@ import { coinbaseWallet, injected } from "wagmi/connectors";
 
 export const config = createConfig({
 	chains: [baseSepolia],
-	connectors: [coinbaseWallet({ appName: "A Cool App" })],
+	connectors: [coinbaseWallet({ appName: "A Cool App", 
+									chainIds: [baseSepolia.id.toString()],
```

Comment:
does it work without this? Ideally it just picks it up from `chains` on L6?

Reply:
i believe for line 6 to work, wagmi will need to update their handling, cuz we changed the config interface in 4.0.0. and line 6 is trying to switch chainId in the 3.9.X way